src/components/molecules/HeroMessage/styles.tsx

```diff
@@ -10,15 +10,14 @@ export const StyleHeroMessageWrapper = styled.div `
     justify-content: space-around;
     max-width: 100%;
     padding: ${space.s} ${space.m} ${space.m} ${space.m};
-    position: relative;
     text-align: center;
 
     ${media.med `
         position: absolute;
-        left: 0;
+        left: 10%;
```

Comment:
why using 10%? This might be a magic number.
https://css-tricks.com/magic-numbers-in-css/

I don't remember the design but if you want to center a position absolute element you can use this:

```
position: absolute;
 top: 50%; 
 right: 50%;
 transform: translate(50%,-50%);
```

Reply:
In the Adobe XD file (desktop version), the element isn't centered. It's slightly off to the left, which is why I used the 10% instead.